packages/react-instantsearch-core/src/connectors/__tests__/connectHits.js

```diff
@@ -11,9 +11,31 @@ describe('connectHits', () => {
     it('provides the current hits to the component', () => {
       const hits = [{}];
       const props = getProvidedProps(null, null, {
-        results: { hits },
+        results: { hits, hitsPerPage: 2, page: 2 },
+      });
+      expect(props).toEqual({
+        hits: hits.map(hit => expect.objectContaining(hit)),
```

Comment:
aren't you just asserting that Array#map works here?

Reply:
😆 You made me double check.
So apparently it works as expected
https://repl.it/repls/SupportiveEveryFirmware

To summarize this checks that `props.hits` and `hits`: 
- are the same length
- contains the same objectIDs as hits, in the same order
```
props.hits = [ { objectID: 1, __position: 1, __queryID: "x" }, { objectID: 2, __position: 1, __queryID: "x" },  ]
hits = [ {objectID: 1}, {objectID: 2} ]
```js

Reply:
ohh I'm seeing the difference now